packages/features/vault/src/components/how-it-works-faq.tsx

```diff
@@ -0,0 +1,164 @@
+                values={{
+                  tokenSymbol,
+                }}
+              />
+            </BodyText>
+            <BodyText>
+              <FormattedMessage
+                defaultMessage={`The vault contract then deploys all the funds into the automated strategy.`}
+              />
+            </BodyText>
+            <ImageWrapper>
+              {'0x05f1ce9c' === vaultStrategyId && (
+                <CurveVaultImg tokenSymbol={tokenSymbol} />
+              )}
+              {'0x77721081' === vaultStrategyId && (
+                <BalancerVaultImg tokenSymbol={tokenSymbol} />
+              )}
+            </ImageWrapper>
+            <H5
+              sx={{
+                color: theme.palette.typography.main,
+                marginBottom: theme.spacing(2),
+              }}
+            >
+              <FormattedMessage defaultMessage={'The Strategy'} />
+            </H5>
+            <BodyText>
+              <FormattedMessage
+                defaultMessage={`1. Take {primaryBorrowCurrency} and provide liquidity to the {primaryBorrowCurrency}/{secondaryCurrency} pool on {baseProtocol}.`}
+                values={{
+                  primaryBorrowCurrency: primaryBorrowCurrency,
+                  secondaryCurrency: secondaryCurrency,
+                  baseProtocol: baseProtocol,
+                }}
+              />
+            </BodyText>
+            <BodyText>
+              <FormattedMessage
+                defaultMessage={`2. Stake the {baseProtocol} LP tokens on {boosterProtocol}.`}
+                values={{
+                  baseProtocol: baseProtocol,
+                  boosterProtocol: boosterProtocol,
+                }}
+              />
+            </BodyText>
+            <BodyText>
+              <FormattedMessage
+                defaultMessage={`3. Harvest and reinvest {incentiveToken2} and {incentiveToken1} incentives back into the pool on a weekly basis.`}
+                values={{
+                  incentiveToken1: incentiveToken1,
+                  incentiveToken2: incentiveToken2,
+                }}
+              />
+            </BodyText>
+            <H5
+              sx={{
+                color: theme.palette.typography.main,
+                marginBottom: theme.spacing(2),
+              }}
+            >
+              <FormattedMessage defaultMessage={'RETURNS'} />
+            </H5>
+            <BodyText>
+              <FormattedMessage
+                defaultMessage={`You earn the strategy APY on your deposit + borrowed funds and you pay the borrow APY on your borrowed funds.`}
+              />
+            </BodyText>
+            <H5
+              sx={{
+                textTransform: 'uppercase',
+                fontSize: '14px',
+                color: theme.palette.typography.light,
+              }}
+            >
+              <FormattedMessage
+                defaultMessage={`Total APY = strategy APY + (strategy APY - borrow APY) * leverage`}
```

Comment:
Captialize this sentence

Reply:
Do you not like textTransform: 'uppercase'?